cmd/mountv1.go

```diff
@@ -194,6 +194,15 @@ var generateConfigCmd = &cobra.Command{
 
 		// Set the endpoint if not explicitly provided
 		if bfv2StorageConfigOptions.Endpoint == "" {
+			accountName := bfv2StorageConfigOptions.AccountName
+			if accountName == "" {
```

Comment:
Container name can also be given on cli param in v1, can we just confirm that part work fine with mountv1